scripts/repos/index.js

```diff
@@ -0,0 +1,117 @@
+const { ethers } = require('ethers')
+const path = require('path')
+const Web3 = require('web3')
+//
+const {
+  encodePublishVersionTxData,
+  getExternalRepoVersion,
+  publishVersion,
+} = require('./apm')
+const { readJson } = require('./utils')
+
+const ensAddress = '0x5f6f7e8cc7346a11ca2def8f827b7a0b612c56a1'
```

Comment:
I would recommend moving any specific param to a file in the root dir that contains all this kind of data in a single place for better visibility and maintainability.